charts/buildkit-service/templates/deployment.yaml

```diff
@@ -93,6 +93,10 @@ spec:
           {{- end }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          {{- with .Values.lifecycle }
```

Comment:
This here is missing a bracket which made the lint job fail.

```suggestion
          {{- with .Values.lifecycle }}
```

Reply:
Oops, thanks for catching this! Fixed via 564ef14